website/content/_includes/layout.njk

```diff
@@ -1,17 +1,18 @@
+{% set base = page.url | relativePathToRoot -%}
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{{ title }} | Wireit</title>
-    <link rel="stylesheet" href="{{ '/style.css' | url }}">
-    <link rel="stylesheet" href="{{ '/prism-light.css' | url }}" media="(prefers-color-scheme:light)">
-    <link rel="stylesheet" href="{{ '/prism-dark.css' | url }}" media="(prefers-color-scheme:dark)">
+    <link rel="stylesheet" href="{{ base }}/style.css">
```

Comment:
No change needed now, but is is possible to write the filter so that the usage is the same as `url`? I find that a bit easier to read and follow.

Reply:
It would look like:

```
{{ 'style.css' | relativePathToRoot(page.url) }}
```

Which I kinda find less readable, personally.